src/NuGetGallery/OData/SearchService/SearchAdaptor.cs

```diff
@@ -25,6 +25,19 @@ public static class SearchAdaptor
         ///     Determines the maximum number of packages returned in a single page of an OData result.
         /// </summary>
         internal const int MaxPageSize = 100;
+        private static IReadOnlyDictionary<string, SortOrder> SortOrders = new Dictionary<string, SortOrder>(StringComparer.OrdinalIgnoreCase)
```

Comment:
Can this be `readonly`?

```suggestion
        private static readonly IReadOnlyDictionary<string, SortOrder> SortOrders = new Dictionary<string, SortOrder>(StringComparer.OrdinalIgnoreCase)
```